tests/integration/helm_fluentbit_fluentd_test.go

```diff
@@ -300,22 +297,23 @@ func Test_Helm_FluentBit_Fluentd(t *testing.T) {
 		Assess("expected container log metadata is present", stepfuncs.WaitUntilExpectedLogsPresent(
 			logsGeneratorCount,
 			map[string]string{
-				"cluster":         "kubernetes",
-				"namespace":       internal.LogsGeneratorName,
-				"pod_labels_app":  internal.LogsGeneratorName,
-				"container":       internal.LogsGeneratorName,
-				"deployment":      internal.LogsGeneratorName,
-				"replicaset":      "",
-				"namespace_id":    "",
-				"pod":             "",
-				"pod_id":          "",
-				"container_id":    "",
-				"host":            "",
-				"master_url":      "",
-				"node":            "",
-				"_sourceName":     "",
-				"_sourceCategory": "",
-				"_sourceHost":     "",
+				"cluster":        internal.ClusterName,
+				"namespace":      internal.LogsGeneratorName,
+				"pod_labels_app": internal.LogsGeneratorName,
+				"container":      internal.LogsGeneratorName,
+				"deployment":     internal.LogsGeneratorName,
+				"replicaset":     fmt.Sprintf("%s%s", internal.LogsGeneratorName, "-[a-z0-9]{9,10}"),
+				"namespace_id":   ".*",
+				"pod":            fmt.Sprintf("%s%s", internal.LogsGeneratorName, internal.PodDeploymentSuffixRegex),
+				"pod_id":         ".*",
+				"container_id":   ".*",
+				"host":           internal.NodeNameRegex,
+				"master_url":     ".*",
+				"node":           internal.NodeNameRegex,
```

Comment:
Does changing from `""` to `".*"` change anything?
IIRC, the receiver mock still supports empty strings as "anything" - is it a preparation for removing this?
If not and you wanted nonempty strings, then `".+"` should be used.

Reply:
Doesn't change anything, but it's clearer this way. But you're right, it should be `.+`, I'll change that.